<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MouseControlWidget</class>
 <widget class="QWidget" name="MouseControlWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>221</width>
    <height>111</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MouseControlWidget</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout_2">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_3">
     <item>
      <widget class="QRadioButton" name="radioButton_Selection">
       <property name="text">
        <string>选择</string>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
       <attribute name="buttonGroup">
        <string notr="true">buttonGroup_3</string>
       </attribute>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="radioButton_Camera_Manipulate">
       <property name="text">
        <string>视角</string>
       </property>
       <property name="shortcut">
        <string>Alt+V</string>
       </property>
       <property name="checked">
        <bool>false</bool>
       </property>
       <attribute name="buttonGroup">
        <string notr="true">buttonGroup_3</string>
       </attribute>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="radioButton_Cutting">
       <property name="text">
        <string>切割</string>
       </property>
       <attribute name="buttonGroup">
        <string notr="true">buttonGroup_3</string>
       </attribute>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="Line" name="line_2">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QStackedWidget" name="stackedWidget">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="page">
      <layout class="QVBoxLayout" name="verticalLayout_6">
       <property name="margin">
        <number>0</number>
       </property>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_2">
           <item>
            <widget class="QRadioButton" name="radioButton_Select_By_Click">
             <property name="text">
              <string>鼠标点选</string>
             </property>
             <property name="checked">
              <bool>true</bool>
             </property>
             <attribute name="buttonGroup">
              <string notr="true">buttonGroup</string>
             </attribute>
            </widget>
           </item>
           <item>
            <widget class="QRadioButton" name="radioButton_Select_By_Rectangle">
             <property name="text">
              <string>长方形框选</string>
             </property>
             <attribute name="buttonGroup">
              <string notr="true">buttonGroup</string>
             </attribute>
            </widget>
           </item>
           <item>
            <widget class="QRadioButton" name="radioButton_Select_By_Polygon">
             <property name="text">
              <string>多边形选</string>
             </property>
             <attribute name="buttonGroup">
              <string notr="true">buttonGroup</string>
             </attribute>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <widget class="Line" name="line">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
          </widget>
         </item>
         <item>
          <layout class="QVBoxLayout" name="verticalLayout">
           <item>
            <widget class="QRadioButton" name="radioButton_Select_Vertices">
             <property name="text">
              <string>点</string>
             </property>
             <attribute name="buttonGroup">
              <string notr="true">buttonGroup_2</string>
             </attribute>
            </widget>
           </item>
           <item>
            <widget class="QRadioButton" name="radioButton_Select_Edges">
             <property name="text">
              <string>边</string>
             </property>
             <property name="checked">
              <bool>true</bool>
             </property>
             <attribute name="buttonGroup">
              <string notr="true">buttonGroup_2</string>
             </attribute>
            </widget>
           </item>
           <item>
            <widget class="QRadioButton" name="radioButton_Select_Faces">
             <property name="text">
              <string>面</string>
             </property>
             <attribute name="buttonGroup">
              <string notr="true">buttonGroup_2</string>
             </attribute>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QPushButton" name="pb_Op_OK">
         <property name="text">
          <string>确定</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="page_2">
      <layout class="QGridLayout" name="gridLayout">
       <property name="margin">
        <number>0</number>
       </property>
       <item row="0" column="0">
        <widget class="QPushButton" name="pushButton">
         <property name="text">
          <string>开始自动旋转</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="page_3">
      <layout class="QVBoxLayout" name="verticalLayout_5">
       <property name="margin">
        <number>0</number>
       </property>
       <item>
        <widget class="QPushButton" name="pushButton_Compute_Cutting">
         <property name="text">
          <string>计算切割</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton_Restore_Cutting">
         <property name="text">
          <string>复原</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
 <buttongroups>
  <buttongroup name="buttonGroup_2"/>
  <buttongroup name="buttonGroup_3"/>
  <buttongroup name="buttonGroup"/>
 </buttongroups>
</ui>
